steps/post-jira-comment-with-build-details/0.1.12/step.yml

```diff
@@ -0,0 +1,104 @@
+title: Create JIRA comment with build
+summary: Will post a comment on the issue page with build details (download link,
+  build number, etc)
+description: Will post a comment on the issue page with build details (download link,
+  build number, etc)
+website: https://github.com/dag-io/post-jira-comment-with-build-details-step
+source_code_url: https://github.com/dag-io/post-jira-comment-with-build-details-step.git
+support_url: https://github.com/dag-io/post-jira-comment-with-build-details-step/issues
+published_at: 2017-05-01T15:37:01.533745086+02:00
+source:
+  git: https://github.com/dag-io/post-jira-comment-with-build-details-step.git
+  commit: 394822b656824386fc5337d04e2f13fbcfe0e49d
+host_os_tags:
+- ubuntu-14.04
+- osx-10.11
+type_tags:
+- jira
```

Comment:
You can keep the other two, it's up to you, `utility` will be important as a "main category" - you'll soon see why this will be important, in the new Workflow Editor ;)